Library/ViewModels/DiscoveryNavigationHeaderViewModel.swift

```diff
@@ -210,7 +210,11 @@ public final class DiscoveryNavigationHeaderViewModel: DiscoveryNavigationHeader
       .map { $0.selected ? Strings.discovery_favorite_categories_buttons_unfavorite_a11y_label() :
         Strings.discovery_favorite_categories_buttons_favorite_a11y_label() }
 
-    self.showFavoriteOnboardingAlert = strings.map { $0.subcategory ?? $0.filter }
+    self.showFavoriteOnboardingAlert = strings
+      .map { $0.subcategory != nil ?
+        Strings.category_name_subcategory_name_saved(
+          category_name: $0.filter, subcategory_name: $0.subcategory ??  "") :  $0.filter
+      }
```

Comment:
I think we should talk about the title for this alert, sorry I didn't see this on the main repo or I would've brought it up then. The string here is "%{category_name}/%{subcategory_name} saved." This means that if there is no subcategory being saved, the copy will read "All Art Projects/ saved." so there would need to be logic to include the "/" character. Even without the slash, "All Art Projects saved." needs to update to capitalize the "saved" text. I think we should regroup with sara on this.